src/WebJobs.Extensions.OpenAI.AzureAISearch/WebJobs.Extensions.OpenAI.AzureAISearch.csproj

```diff
@@ -2,18 +2,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure WebJobs SDK Extension for OpenAI - Azure AI Search Package.</Description>
-	</PropertyGroup>
-	<!-- Version settings: https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/ -->
-	<PropertyGroup>
-		<MajorVersion>0</MajorVersion>
-		<MinorVersion>1</MinorVersion>
-		<PatchVersion>0</PatchVersion>
-		<VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
-		<VersionSuffix>alpha</VersionSuffix>
-		<AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>
-		<FileVersion>$(VersionPrefix).0</FileVersion>
-		<!-- FileVersionRevision is expected to be set by the CI. This is useful for distinguishing between multiple builds of the same version. -->
-		<FileVersion Condition="'$(FileVersionRevision)' != ''">$(VersionPrefix).$(FileVersionRevision)</FileVersion>
+    <Version>$(AzureAISearchVersion)</Version>
```

Comment:
Looks like this file has a mix of spaces and tabs. Can you convert all the tabs into spaces for consistency?